style.css

```diff
@@ -47,16 +47,39 @@ main {
   background-color: #f7f7f9;
 }
 
-.introduction {
-  background-image: url(./images/header-bg.png);
-  background-repeat: no-repeat;
+.mobile-nav {
+  position: absolute;
```

Comment:
### To Improve and it's really just an opinion 😁

When I click the menu button am still allowed to scroll down to the home page so I highly recommend preventing the user from scrolling when the menu is open, to achieve that I suggest these two options:

1. You can create a class that applies `overflow: hidden;` to the body tag and just toggle that class when the menu button is clicked using `javascript`.
2. Or the easy route is using `position: fixed` instead of `position: absolute` to the `.mobile-nav`, but be aware of the scrolling will keep functioning, but it will look just fine 👌

Reply:
done

Reply:
@adel-gu I have made all the requested changes